workshop/exercise-3/README.md

```diff
@@ -1,57 +1,82 @@
 ## Deploy our First Application to Knative using the Knative Client (kn)
-The knative client aims to make interacting with Knative a seamless experience for developers. We'll try deploying an application using `kn`, then we'll deploy the same application using `kubectl` and `.yaml` files for additional control. The application for this lab is a simple Node.js with Express app which returns the first n numbers of the Fibonacci sequence. Once the app is deployed, you can use it by making a GET request to the `/` endpoint with a number as the parameter. 
+The Knative client, `kn`, aims to make interacting with Knative a seamless experience for developers. In this section we'll try deploying an application using `kn`. The application for this lab is a simple Node.js with Express app which returns the first n numbers of the Fibonacci sequence. Once the app is deployed, you can use it by making a GET request to the `/` endpoint with a number as the parameter. 
 
-# Our First Knative Service
-Knative Serving supports deploying and serving of serverless applications. Those applications will automatically scale up, and then back down to zero. In this exercise, we'll use the Knative Serving component to deploy our first application from a container image hosted on dockerhub. 
+### Our First Knative Service
+Knative Serving enables rapid deploying and serving of serverless applications. Those applications will automatically scale up, and then back down to zero. In this exercise, we'll use the Knative Serving component to deploy our first application from a container image hosted on dockerhub. 
 
-The Knative Service object automatically manages the whole lifecycle for your workload. A service represents your app on Knative. Services control the creation of other objects to ensure that your app has a URL and a new revision for each update of the service.
+The Knative Service object automatically manages the whole lifecycle for your workload. A Service represents your app on Knative. Services control the creation of other objects to ensure that your app has a URL and a new Revision for each update of the service.
 
-![](https://github.com/knative/serving/raw/master/docs/spec/images/object_model.png)
+![](https://github.com/knative/serving/raw/master/docs/spec/images/fibknativev1.png)
```

Comment:
I think the URL for this is wrong - git diff shows it at:  workshop/README_images/fibknativev1.png
Might want to check the other images too

Reply:
i think this link should work -- i'll confirm once we merge since the images are a part of this PR.